nupic/research/frameworks/pytorch/model_utils.py

```diff
@@ -36,11 +36,23 @@ def train_model(
     optimizer,
     device,
     freeze_params=None,
+    freeze_fun=None,
+    freeze_pct=90,
+    freeze_output=False,
```

Comment:
Looks like these changes are still in the PR? You should take these out.

Reply:
right so i wasn't sure here since it had also been suggested that i keep it in model_utils.py since most of the code is the same. will remove "freeze_output_layer" though

Reply:
No, by "not change this function" I meant don't change it at all in this PR since these changes are very specific to your work.  Create a separate function (which you did) and use that. In this PR there would then be no changes at all to model_utils.py (which is used by everyone and thus should be treated carefully).